src/subscription/subscription.service.ts

```diff
@@ -24,12 +25,36 @@ export class SubscriptionService {
     return true;
   }
 
-  public async getActivities(tripType: string, after: Date) {
+  // TODO move to activity module?
+  public async getNewActivities(tripType: string, after: Date) {
     return this.dbService.prisma.activities.findMany({
       where: {
-        tripType: { has: tripType },
+        tripType: { has: tripType }, // TODO make this case INsensitive! https://www.prisma.io/docs/concepts/components/prisma-client/case-sensitivity
         createdAt: { gt: after },
       },
     });
   }
+
+  // TODO move to cabins module?
```

Comment:
Was wondering about that. I am in two minds. One approach is having each module describing its own queries and another is  having all the queries centralised.

I think I like the first one for now as it makes all the module more isolated.

Reply:
This was exactly my thought as well and I agree that it would be nice to keep it within each module, then it is easier to see if such a method already exists if we ever want to get new cabins for example outside of the subscription feature.